OrrRock/OrrRock/VideoDetail/VideoCollection/VideoCollectionView+VideoCollectionViewExtensions.swift

```diff
@@ -89,23 +89,23 @@ extension VideoCollectionViewController :  UICollectionViewDelegate{
 
 extension VideoCollectionViewController  : UICollectionViewDataSource{
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let screenBounds = UIScreen.main.bounds
-        let screenScale = UIScreen.main.scale
-        let screenSize = CGSize(width: screenBounds.size.width * screenScale, height: screenBounds.size.height * screenScale)
+
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "customVideoCollectionCell", for: indexPath) as! VideoCollectionViewCell
-        cell.cellImage.image = videoInformationArray[indexPath.row].videoLocalIdentifier?.generateCardViewThumbnail(targetSize: screenSize )
+        
+        cell.cellImage.image = videoInformationArray[indexPath.row].videoLocalIdentifier?.generateCardViewThumbnail()
```

Comment:
```suggestion
        cell.cellImage.image = videoInformationArray[indexPath.row].videoLocalIdentifier?.generateCardViewThumbnail(targetSize: cell.contentView.frame.size)
```

나머지 부분은 수정하지 않고, 필요한 이미지에 비해 10배 가량 사이즈가 큰 이미지를 호출하는 해당 size만 딱 필요한 사이즈로 호출하도록 수정하면 될 것 같아요.

Reply:
혹은 이미지의 targetSize가 얼마로 설정되든 상관없이 이미지가 nil이 아닌 값으로 들어오는 것이 일단 더 중요하다 라고 판단된다면, 이미지는 asset.pixelWidth, asset.pixelHeight 값을 활용해 항상 원본 사이즈로 불러오고, 로드된 UIImage를 원하는 크기로 resizing 하여 return 하는 방법도 충분히 고려해볼만 한 것 같습니다.

Reply:
현재 방식에서는 view의 thumbnail을 `requestImage`를 통해 하나씩 불러오는 형태인 것 같은데 사용자가 뷰를 여러 번 출입하는 상황을 고려하거나 Collection View의 데이터를 전체적으로 불러와야한다는 점에서는 이미지를 한 번에 Caching하는 [PHImageCachingManager](https://developer.apple.com/documentation/photokit/phcachingimagemanager)로 뷰 전체 이미지를 한 번에 불러와서 대기시키는 것도 좋은 방법이 될 것 같아요!